db/yaml/workspaces/jupyterlab/20210323175655.yaml

```diff
@@ -0,0 +1,101 @@
+containers:
+  - name: jupyterlab
+    image: onepanel/dl:0.17.0
```

Comment:
This should be upgraded to onepanel/dl:0.20.0

Reply:
Done.